rust-runtime/aws-smithy-http-server/examples/pokemon-service/src/main.rs

```diff
@@ -47,10 +52,21 @@ pub async fn main() {
         // implementation.
         .into();
 
+    // New builder
```

Comment:
I think it's best we don't modify the public Pokémon Service example until we're ready to move people to the new builder and deprecate the old one.

We should instead create a Kotlin unit test that generates a simple model and uses the new builder instead. It could also assert that the new builder is generated as doc hidden in `lib.rs`.